ocaml/xapi/xapi_pool.ml

```diff
@@ -1861,3 +1861,9 @@ let add_to_guest_agent_config ~__context ~self ~key ~value =
 let remove_from_guest_agent_config ~__context ~self ~key =
   Db.Pool.remove_from_guest_agent_config ~__context ~self ~key;
   Xapi_pool_helpers.apply_guest_agent_config ~__context
+
+let set_default_SR ~__context ~self ~sr =
+  Db.Pool.set_default_SR ~__context ~self ~value:sr;
+  if Db.SR.get_shared ~__context ~self:sr then begin
```

Comment:
When someone changes the default SR from a shared SR to a non-shared SR, have we thought about whether we should disable the redo log?